src/Test/HttpClient/MockHttpAdapter.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace FOS\HttpCache\Test\HttpClient;
+
+use Http\Adapter\Exception\MultiHttpAdapterException;
+use Http\Adapter\HttpAdapter;
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+
+class MockHttpAdapter implements HttpAdapter
```

Comment:
and does php-http not provide something like that? we might contribute it there actually - this has nothing to do with cache invalidation...